static/sass/_snapcraft_market.scss

```diff
@@ -116,4 +118,82 @@
       line-height: 2.3rem;
     }
   }
+
+  .p-market-banner {
+
+    .p-market-banner__image-holder {
+      cursor: pointer;
+      position: relative;
+    }
+
+    .p-market-banner__image {
+      background-color: $color-x-light;
+      background-position: center;
+      background-size: cover;
+      border: 1px solid $color-mid-light;
+      border-radius: $border-radius;
+      color: $color-link;
+      cursor: pointer;
+      height: 128px;
+      margin-bottom: $spv-inter--regular;
+      position: relative;
+      text-align: center;
+    }
+
+    .p-market-banner__image-holder:hover,
+    .p-market-banner__image-holder.is-focused {
+      .p-market-banner__image {
+        border: 1px solid $color-link;
+      }
+
+      .p-market-banner__remove {
+        opacity: 1;
+      }
+
+      .p-market-banner__change {
+        border: 1px solid $color-link;
+        border-radius: $border-radius;
+        opacity: 1;
+      }
+    }
+
+    .p-market-banner__remove {
+      background: $color-x-light;
+      cursor: pointer;
+      opacity: 0;
+      padding: .25rem;
+      position: absolute;
+      right: 1px;
+      top: 1px;
+      z-index: 1;
+
+      .p-icon--delete {
+        display: block;
+        top: 0;
+      }
+    }
+
+    .p-market-banner__change {
+      align-items: center;
+      background-color: $white-overlay-strong-color;
+      bottom: 0;
+      color: $color-link;
+      display: flex;
+      justify-content: center;
+      left: 0;
+      opacity: 0;
+      pointer-events: none;
+      position: absolute;
+      right: 0;
+      top: 0;
+    }
+  }
+
+  .p-icon--delete {
+    @include snapcraft-icon-delete($color-mid-dark);
+
+    &:hover {
```

Comment:
Icon only changes colour when hovering over icon itself, not when hovering over whole clickable "remove" area.

<img width="757" alt="Screenshot 2019-04-09 at 08 31 59" src="https://user-images.githubusercontent.com/83575/55778037-111fa880-5aa2-11e9-9c49-817dc8eaa9b2.png">